### ISSUE TITLE
Messages disappear after being sent | Bug

### ISSUE THREAD

**Issue by ulugmer** [Ajaxy/telegram-tt#320]
This video from ulu, but Telegram A has same issue
https://www.loom.com/share/fa73f462f826424ca1151b28f8cb8915?sid=519c2a07-44cf-4487-a597-770d08aa32ab

Please fix it 🙏

**Comment by ingria** [comment]
I have the same problem too (Firefox x64)

**Comment by Ajaxy** [comment]
Is this specifically related to forum topics?

**Comment by ulugmer** [comment]
> Is this specifically related to forum topics?

No, it is present everywhere, but reproduction is easier in Topics.

**Comment by Ajaxy** [comment]
Can you record a video with the reproduction in a regular chat? @ulugmer

**Comment by ulugmer** [comment]
I have tried reproducing it in both groups and chats, but unfortunately (or fortunately), I can't. It seems that this issue only occurs in forums.

**Comment by Ajaxy** [comment]
Can you still reproduce that with v10.0.24?

**Comment by Ajaxy** [comment]
Should we close this?

**Comment by ulugmer** [comment]
> Can you still reproduce that with v10.0.24?

Yes, I can. And in v10.0.25 too
https://www.loom.com/share/3d2c0385c8b94c17a8b78ba1981c3348?sid=0abb3838-12f8-49b2-a10c-ef73eef09668
